server/registry/internal/storage/models/api.go

```diff
@@ -37,6 +37,8 @@ type Api struct {
 	RecommendedDeployment string    // Recommended API deployment.
 	Labels                []byte    // Serialized labels.
 	Annotations           []byte    // Serialized annotations.
+	ParentKey             string
+	Parent                *Project `gorm:"foreignKey:ParentKey;constraint:OnUpdate:CASCADE,OnDelete:CASCADE"`
```

Comment:
Could we add these keys to an existing database with a database migration? I think that would require something more than the default gorm migration that we're using in the `AdminService` `migrateDatabase` RPC.

Reply:
Unsure. I'll have to figure out a test for that.